tools/cli/adminElasticSearchCommands.go

```diff
@@ -231,3 +256,233 @@ func generateESDoc(msg *indexer.Message) map[string]interface{} {
 	}
 	return doc
 }
+
+// This function is used to trim unnecessary tag in returned json for table header
+func bucketKey(k string) string {
+	// group key is in form of "group_key", we only need "key" as the column name
+	if strings.HasPrefix(k, "group_") {
+		k = k[6:]
+	}
+	if strings.HasPrefix(k, "Attr_") {
+		k = k[5:]
+	}
+	return fmt.Sprintf(`%v(*)`, k)
+}
+
+// parse the returned time to readable string if time is in int64 format
+func timeStr(s interface{}) (string, error) {
```

Comment:
rename to `toTimeStr`, otherwise mixed name with some local var. 

It usually a good practice to name function with something like verb+noun